packages/firebase_ml_vision/example/lib/main.dart

```diff
@@ -2,212 +2,49 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:async';
-import 'dart:io';
-
-import 'package:firebase_ml_vision/firebase_ml_vision.dart';
 import 'package:flutter/material.dart';
-import 'package:image_picker/image_picker.dart';
-
-import 'detector_painters.dart';
 
-void main() => runApp(MaterialApp(home: _MyHomePage()));
+import 'picture_scanner.dart';
+
+void main() {
+  runApp(
+    MaterialApp(
+      routes: <String, Widget Function(BuildContext)>{
+        '/': (BuildContext context) => _ExampleList(),
+        '/$PictureScanner': (BuildContext context) => PictureScanner(),
+      },
+    ),
+  );
+}
 
-class _MyHomePage extends StatefulWidget {
```

Comment:
This widget was moved to `picture_scanner.dart` and renamed to `PictureScanner`.